index_website.html

```diff
@@ -0,0 +1,418 @@
+    <link rel="stylesheet" href="styles_website.css">
+</head>
+<body>
+  <div class="site">
+
+
+  <div class="page1">  
+    <div class="menu1">
+      <div class="logo1">
+          <img src="img/Logo.png" alt="r">
+        </div>
+          <div class="links1_1">
+            <ul>
+              <li><a href="">How It Works</a></li>
+              <li><a href="">Help</a></li>
+              <li><a href="">Payment</a></li>
+              <li><a href="">Fees</a></li>
+            </ul>
+          </div>
+        <div class="relative">
+          <div class="menu1_2">
+             <a href="url">Sign in</a>
+          </div>
+        <div class="menu1_3">
+          <button type="button" name="show_more">Sign Up</button>
+        </div>
+      </div>
+    </div>
+    
+  <div class="page2">
+    <div  class="block1">
+      <div class="text">
+        <div class="head1">
+          <h1>
+            Balance Your Best Intersts
+          </h1>
+        </div>
+      <div class="paragraph1">
+        <p>
+          It is a long established fact that a reader will be distracted 
+          by the readable content of a page when looking at its layout.
+        </p>
+      </div>
+    </div>
+  <div class="relative2">
+    <div class="menu2_1">
+      <button type="button" name="show_more">Create An Account</button>
+    </div>
+      <div class="picture1">
+        <img src="img/IMG_5862.PNG">
+      </div>
+        <div class="paragraph2">
+          <p>
+            See How It Works?
+          </p>
+        </div>
+      </div>
+    </div>
+      <div class="picture2">
+        <img src="img/Illustrations.png" alt="">
+      </div>
+  </div>
+ </div>
+
+  <div class="page3">
+    <div class="top_text">
+      <div class="head3_1">
+        <h1>
+          Why You Choose Us?
+        </h1>
+      </div>
+    <div class="paragraph3_1">
+      <p>
+        There are many variations of passages of Lorem Ipsum available,
+         but the majority have suffered 
+      </p>
+    </div>
+  </div>
+  <div class="column">
+    <div class="column1">
+      <div class="picture3_1">
+        <img src="img/Illustrations 01.png" alt="">
+      </div>
+   <div class="head3_2">
+    <h1>
+      Easy To Use
+    </h1>
+  </div>
+    <div class="paragraph3_2">
+      <p>
+        There are many variations of passages of lorem
+        Ipsum available, but the majority have suffered alteration in some form
+      </p>
+    </div>
+  </div>
+
+<div class="column2">
+  <div class="picture3_2">
+    <img src="img/Illustration 02.png" alt="">
+  </div>
+
+  <div class="head3_3">
+    <h1>
+      Faster Payments
+    </h1>
+  </div>
+
+  <div class="paragraph3_3">
+    <p>
+      There are many variations of passages of lorem
+      Ipsum available, but the majority have suffered alteration in some form
+    </p>
+  </div>
+</div>
+
+<div class="column3">
+  <div class="picture3_3">
+    <img src="img/Illustration 03.png" alt="">
```

Comment:
why do you use empty spaces in the name of the file?

what does mean illustration? rename pls